bpf/lib/process.h

```diff
@@ -264,6 +246,8 @@ struct msg_execve_event {
 	struct msg_execve_key parent;
 	__u64 parent_flags;
 	struct msg_capabilities caps;
+	/* TODO: convert this tg_cred_minimal to tg_cred and include caps above inside */
+	struct tg_cred_minimal creds;
```

Comment:
Why not keep the naming consistent here (i.e. have the `msg_` prefix)? For example `msg_cred_minimal`. It's better to be consistent here. 

Same for `tg_user_namespace` and `tg_cred`.

Reply:
Alright , done ;-)